Library/Formula/bazel.rb

```diff
@@ -9,6 +9,7 @@ class Bazel < Formula
     sha256 "fe665d9ff99da2a40531106ad78cbe9ac41bfeacc17822280960aa86eea23a01" => :el_capitan
     sha256 "e1cefe27d4eaa65a42578d0872b6936bb0670d03f013e790da4dbda27449c6b1" => :yosemite
     sha256 "333a3699765ced173ece8d9f355d9cba8b9007de4a67c5dacb88511ec3bc7774" => :mavericks
+    revision 1
```

Comment:
You can use `brew install -s bazel` if you want to force it to install from source instead of pouring a bottle. `-s` is short for `--build-from-source`.